### PR TITLE
Send tests results to Sheet

### LINKED ISSUE #6451: Flakes analysis on IBM cloud tests
/kind tests

we need some way to analyse the flakes observed as IBM Clouds tests to make tests more reliable  

https://github.com/kadel/odo-tools fetches logs from https://search.ci.openshift.org, but since we stopped using Prow, this tool can be modified to fetch logs from IBM cloud.

### PR DESCRIPTION
**Pull request by feloy**
**What type of PR is this:**

/kind feature
/area testing

**What does this PR do / why we need it:**

Send data collected into a junit file during integration tests to a Goole Sheet

**Which issue(s) this PR fixes:**

Fixes #6451 

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

File: .ibm/tools/tests-results/go.mod [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module github.com/feloy/junit-collector
```
Comment (rm3l) [inline .ibm/tools/tests-results/go.mod:1]: Can we change the module to something like `github.com/redhat-developer/junit-collector`?

File: .ibm/tools/tests-results/main.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	logFile := *fLogFile
+
+	ctx := context.Background()
+
+	oldDate := time.Now().Add(-15 * 24 * time.Hour)
+	fmt.Printf("Deleting entries before: %s\n", oldDate.Format("2006-01-02"))
+	err := db.Clean(ctx, spreadsheetId, oldDate.Format("2006-01-02"))
+	if err != nil {
+		panic(err)
+	}
```
Comment (valaparthvi) [inline .ibm/tools/tests-results/main.go:69]: This can now be removed. An automation task has been setup to clean 15 days old data every night.

**Comment by valaparthvi** [comment]
Is it possible to also set this up for podman tests running via GH actions.

**Review by valaparthvi — APPROVED** [review]

**Comment by feloy** [comment]
/override Kubernetes-Integration-Tests/Kubernetes-Docs-Integration-Tests
Known issue (#6876)

**Comment by feloy** [comment]
/override OpenShift-Integration-tests/OpenShift-Integration-tests
Unrelated

**Comment by feloy** [comment]
/override "SonarCloud Code Analysis"
Code duplication on mocks

**Pull request merged**